daphne/src/protocol/aggregator.rs

```diff
@@ -1,6 +1,3 @@
-// Copyright (c) 2024 Cloudflare, Inc. All rights reserved.
-// SPDX-License-Identifier: BSD-3-Clause
-
```

Comment:
Why remove copywrite?

Reply:
Whoopsie daisy, no idea how that happened. Will restore.